virtualenv.py

```diff
@@ -1185,26 +1185,55 @@ def change_prefix(filename, dst_prefix):
     assert False, "Filename {} does not start with any of these prefixes: {}".format(filename, prefixes)
 
 
-def copy_required_modules(dst_prefix, symlink):
-    import warnings
+if sys.version_info < (3, 4):
+    import imp
+
+    def find_module_filename(modname):
+        try:
+            f, filename, _ = imp.find_module(modname)
```

Comment:
I would prefer a more verbose name than f, maybe ``file_handler``?

Reply:
this is existing code -- I just moved it

Reply:
Indeed, :-D if you don't mind changing it though that would be great.

Reply:
I can if you want, but then it won't preserve blame:

```
$ git blame -w -M3 virtualenv.py | grep 'f, filename'
d777eb63 virtualenv.py     (Antonio Cuni           2010-06-30 14:57:52 +0200 1193)             f, filename, _ = imp.find_module(modname)
```

Reply:
I'm alright here with as we're talking about a low number of trivial lines.

Reply:
idk about you but I'd rather not have the deprecated lines blame to me ;)

Reply:
fair enough

Reply:
lol rip, they ended up blaming to me anyway :'(

Reply:
I shelter the blame, feel free to blame me extra. 👍